benchmark_runner/workloads/bootstorm_vm.py

```diff
@@ -143,7 +157,7 @@ def _run_vm(self):
         self._set_bootstorm_vm_first_run_time()
         self._set_bootstorm_vm_start_time(vm_name=self._vm_name)
         self._virtctl.start_vm_sync(vm_name=self._vm_name)
-        self.vm_node = self._wait_ssh_vm(vm_name=self._vm_name)
+        self.vm_node = self._wait_virtctl_vm(vm_name=self._vm_name)
```

Comment:
Change to `_wait_for_vm_status` or similar without specifying the mechanism in the name.

Reply:
I prefer to use the right term virtctl instead of status.
If there is any issue, it will be more informative.

Reply:
OK.